src/main/java/lotto/domain/WinningInformation.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class WinningInformation {
+
+    private static final LinkedHashMap<WinningPrize, Integer> prizeAndCount = new LinkedHashMap<>();
```

Comment:
```suggestion
    private static final Map<WinningPrize, Integer> prizeAndCount = new LinkedHashMap<>();
```

추상화된 interface로 받는 것돠 구체화된 타입을 받는 것은 어떤 차이가 있을까요?

DIP에 대해 알아보아요!
https://inpa.tistory.com/entry/OOP-%F0%9F%92%A0-%EC%95%84%EC%A3%BC-%EC%89%BD%EA%B2%8C-%EC%9D%B4%ED%95%B4%ED%95%98%EB%8A%94-DIP-%EC%9D%98%EC%A1%B4-%EC%97%AD%EC%A0%84-%EC%9B%90%EC%B9%99